src/status_im/subs/chat/chats.cljs

```diff
@@ -385,18 +385,22 @@
  (fn [[current-chat pk]]
    (group-chat/member-removed? current-chat pk)))
 
+(defn compute-mentionable-users
+  [{:keys [users community-id] :as chat} blocked all-contacts
+   {:keys [public-key] :as current-multiaccount}]
+  (let [community-members @(re-frame/subscribe [:communities/community-members community-id])
```

Comment:
subscription still used in event, it's better probably to try to pass :chats/mentionable-users value into :chat.ui.input/set-chat-input-text from the UI

Reply:
you right. I did not notice